src/utils/string.js

```diff
@@ -0,0 +1,3 @@
+module.exports.spliceURL = function spliceURL() {
+  return Array.from(arguments).reduce((lastResult, current) => lastResult + current).replace(/([^:])(\/)+(.)/g, `$1/$3`);
```

Comment:
Why use arguments instead normal parameters?

Reply:
Unlimited number of arguments, `spliceURL('https://example.org', '/future/', 'use')`

Reply:
I see, Now I remember you wanted to use spread operator.

Reply:
Could you rebase ?

Reply:
Done